docs/make.jl

```diff
@@ -66,30 +67,32 @@ end
 
 makedocs(
     modules = [RigidBodyDynamics, RigidBodyDynamics.OdeIntegrators],
-    root = @__DIR__,
+    format = Documenter.HTML(),
     checkdocs = :exports,
-    sitename ="RigidBodyDynamics.jl",
+    sitename = "RigidBodyDynamics.jl",
     authors = "Twan Koolen and contributors.",
     pages = [
         "Home" => "index.md",
         "Tutorials" => tutorialpages,
         "Library" => [
-            "Spatial vector algebra" => "spatial.md",
+            "Spatial Vector Algebra" => "spatial.md",
             "Joints" => "joints.md",
-            "Rigid bodies" => "rigidbody.md",
+            "Rigid Bodies" => "rigidbody.md",
             "Mechanism" => "mechanism.md",
-            "MechanismState" => "mechanismstate.md",
-            "Kinematics/dynamics algorithms" => "algorithms.md",
-            "Custom collection types" => "customcollections.md",
-            "Cache types" => "caches.md",
+            "Mechanism State" => "mechanismstate.md",
+            "Kinematics/Dynamics Algorithms" => "algorithms.md",
+            "Custom Collection Types" => "customcollections.md",
+            "Cache Types" => "caches.md",
             "Simulation" => "simulation.md",
-            "URDF parsing and writing" => "urdf.md",
+            "URDF Parsing and Writing" => "urdf.md",
         ],
         "Benchmarks" => "benchmarks.md",
-      ],
-    format = Documenter.HTML(prettyurls = parse(Bool, get(ENV, "CI", "false")))
+    ],
+    warnonly = Documenter.except(),
 )
 
 deploydocs(
-    repo = "github.com/JuliaRobotics/RigidBodyDynamics.jl.git"
+    repo="github.com/JuliaRobotics/RigidBodyDynamics.jl",
+    devbranch="master",
+    push_preview=true,
```

Comment:
`push_preview = true` turned out to be useless for this PR because it only works for PRs originating from the same repo instead of from forked repositories — see [this line of the job](https://github.com/JuliaRobotics/RigidBodyDynamics.jl/actions/runs/11757946829/job/32755785129?pr=648#step:5:471).